builder/builder.go

```diff
@@ -1760,6 +1798,38 @@ func (b *Builder) BuildImage(format string, template string) error {
 	return imagecmd.Run()
 }
 
+// UpdateImageBundles will update the image bundle list based on the mix bundles.
+// If there is an error reading the mix bundles the default bundle list will be used.
+func UpdateImageBundles(configValues *[]byte, mixBundlesFile string) error {
+	mixBundles, err := helpers.ReadFileAndSplit(mixBundlesFile)
+	if err != nil {
+		return err
+	}
+
+	var bundles []string
+	for _, bundle := range mixBundles {
+		bundle = strings.TrimSpace(bundle)
+		if bundle != "" {
+			bundles = append(bundles, bundle)
+		}
+	}
```

Comment:
I considered using this method. But it gets additional bundle info and not just the name. So to avoid unnecessary additional data I decided not to use it.

Reply:
I'd rather used the well-defined way to get the bundle list. If you need something more stripped-down define it as a function on on `builder` like `getMixBundlesListAsSet` is.

I really don't want to add manual parsing of a file in random parts of the codebase. It is very unmaintainable if we end up changing the file format in the future.

Reply:
Done. Used existing b.getMixBundlesListAsSet()